src/main/kotlin/solve/main/MainView.kt

```diff
@@ -1,14 +1,55 @@
 package solve.main
 
-import solve.catalogue.view.CatalogueView
+import javafx.scene.control.SplitPane
 import solve.menubar.view.MenuBarView
 import solve.scene.view.SceneView
+import solve.sidepanel.content.SidePanelContentView
+import solve.sidepanel.SidePanelTabsView
+import solve.utils.createPxBox
 import tornadofx.*
 
 class MainView : View() {
+    companion object {
+        private const val SideAndSceneDividerPosition = 0.25
+    }
+
+    private val sceneView: SceneView by inject()
+    private val sidePanelContentView: SidePanelContentView by inject()
+
+    private lateinit var mainSplitPane: SplitPane
+    private var lastSideAndSceneDividerPosition = SideAndSceneDividerPosition
+
     override val root = borderpane {
         top<MenuBarView>()
-        center<SceneView>()
-        left<CatalogueView>()
+        mainSplitPane = splitpane {
+            addStylesheet(MainSplitPaneStyle::class)
+            add(sidePanelContentView)
+            add(sceneView)
+            SplitPane.setResizableWithParent(sidePanelContentView.root, false)
```

Comment:
Why false, explain me please how width ratio is configured here? If it is done in components, it is bad approach, it should be configured in main view

Reply:
When I resize window catalogue takes almost all space, it is not good. Is there any way to preserve ratio?
![idea64_mIBJOlsil0](https://user-images.githubusercontent.com/55746901/209585889-f1e27259-0da5-4a9e-a75a-8a36f8ba8705.gif)

Reply:
> Why false, explain me please how width ratio is configured here? If it is done in components, it is bad approach, it should be configured in main view

This is false because it is needed for saving ratio when resizing the window.

Reply:
> When I resize window catalogue takes almost all space, it is not good. Is there any way to preserve ratio? 

Fixed
![resize](https://user-images.githubusercontent.com/66365072/210097247-c48b8fd7-832c-4f85-8ce5-afcdd009c1bf.gif)